e2etest/local_transfer_test.go

```diff
@@ -157,6 +157,15 @@ var (
 			TsfSuccess, "",
 			"Transfer with same nonce from a single sender 1",
 		},
+		{
+			AcntCreate, nil, big.NewInt(100000),
+			AcntCreate, nil, big.NewInt(100000),
+			0, big.NewInt(0),
+			make([]byte, 4),
+			uint64(200000), big.NewInt(1),
+			TsfSuccess, "",
+			"Transfer with nonce 0",
```

Comment:
Why behavior changes when this feature not enabled

Reply:
the test is "transfer with nonce 0". Before enable this, legacy account first tx has `nonce=1`, so the test will fail. Now with this feature enabled, the first tx can have `nonce=0` so the test can pass